src/main/java/emu/grasscutter/server/game/GameServerInitializer.java

```diff
@@ -13,8 +13,6 @@ public GameServerInitializer(GameServer server) {
 	
     @Override
     protected void initChannel(UkcpChannel ch) throws Exception {
-        ChannelPipeline pipeline = ch.pipeline();
-        GameSession session = new GameSession(server);
-        pipeline.addLast(session);
+        new GameSession(server,ch.pipeline());
```

Comment:
Please keep the existing code as is, in case a session needs to be created without a pipeline.